modules/_node-scripts/util/launchSoffice.mjs

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {$} from 'execa';
+
+import fileExists from './fileExists.mjs';
+
+export default async function launchSoffice(...args) {
+	if (await fileExists('/usr/bin/soffice')) {
+		await $`/usr/bin/soffice ${args.join(' ')}`;
```

Comment:
idk if this is necessary, might be overkill to launch an app and instead just log the location of the file.

Reply:
well, I added it because I was tired of generating the file, opening a browser, double clicking and all that :sweat_smile: 

But now, thinking about it... If we run this in the CI that may break things and now that it's completely developed I must admit that that browsing for files back and forth shouldn't happen any more so I'll remove it :+1:

Reply:
I removed it.